src/components/Navigation/Topics.jsx

```diff
@@ -0,0 +1,28 @@
+import { Link } from "react-router-dom";
+
+export default function Topics({ setTopic }) {
+  return (
+    <div>
+      <ul>
+        <li className="topics">
+          {" "}
+          <Link to="/?topic=coding">
```

Comment:
if new topics were added we would still only have these three topics. 
Its worth doing an api call to get the topics available :)